scripts/outer/kubectl-kubestellar-prep_for_syncer

```diff
@@ -151,41 +161,36 @@ else
 fi
 
 if ! kubectl "${kubectl_flags[@]}" get Workspace "$mbwsname" &> /dev/null; then
-    sleep 15 # maybe the mailbox controller is slow, give it a chance
+	echo "maybe the mailbox controller is slow, give it a chance"
+    sleep 15
 fi
 
 if ! kubectl "${kubectl_flags[@]}" get Workspace "$mbwsname" &> /dev/null; then
-    sleep 15 # maybe the mailbox controller is slow, give it another chance
+	echo "maybe the mailbox controller is slow, give it another chance"
+    sleep 15
 fi
 
 if ! kubectl "${kubectl_flags[@]}" get Workspace "$mbwsname" &> /dev/null; then
     echo "$0: did not find the mailbox workspace $mbwsname; is the mailbox controller running?" >&2
     exit 5
 fi
 
-# Now that workspace _exists_, but is it _ready_?
+echo "now that workspace _exists_, but is it _ready_?"
 sleep 5
 
 if ! kubectl ws "${kubectl_flags[@]}" "$mbwsname" &> /dev/null; then
+	kubectl ws .
+	echo "tried once to switch to $mbwsname"
     sleep 15
     if ! kubectl ws "${kubectl_flags[@]}" "$mbwsname" &> /dev/null; then
+	kubectl ws .
+	echo "tried twice to switch to $mbwsname"
 	sleep 15
+	kubectl ws .
 	kubectl ws "${kubectl_flags[@]}" "$mbwsname"
     fi
 fi
 
-# Now the MBWS exists and is ready, but does it have the needed APIBinding?
-
-if ! kubectl "${kubectl_flags[@]}" get APIBinding bind-edge &> /dev/null; then
-    sleep 15
-    if ! kubectl "${kubectl_flags[@]}" get APIBinding bind-edge &> /dev/null; then
-	sleep 15
-	kubectl "${kubectl_flags[@]}" get APIBinding bind-edge > /dev/null
-    fi
-fi
-
-# Now the needed APIBinding exists, has it taken effect?
-sleep 10 # hope so
-
-
-"$bindir/kubectl-kubestellar-syncer_gen" "$stname" --syncer-image "$syncer_image" -o "$output"
+echo "what is the workspace now?"
+kubectl ws .
```

Comment:
If you really want this debugging, make it print one line.

```suggestion
echo "Current workspace is now $(kubectl ws "${kubectl_flags[@]}" . --short)"
```

Reply:
Changed as shown.

Reply:
done